ports/geckolib/properties.mako.rs

```diff
@@ -354,14 +358,20 @@ impl Debug for ${style_struct.gecko_ffi_name} {
    # These are booleans.
    force_stub += ["page-break-after", "page-break-before"]
 
-   simple_types = ["Number", "Opacity"]
+   # Properties that would be auto-generated but need to be manually implemented.
+   manual = ["bottom", "left", "right", "top"]
```

Comment:
I don't understand why we need this - doesn't this duplicate the purpose of the skip_longhands annotation that we have on the style struct declarations?